src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,107 @@
+.page {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: $main-color;
+}
+
+.product-card {
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+
+  &__product-image {
+    height: 134px;
+    margin: 16px 0 40px;
+    padding-inline: 3px;
+    background-image: url(../images/imac.jpeg);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__product-title {
+    text-align: left;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0;
+  }
+
+  &__product-code {
+    margin: 4px 0 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: $secondary-color;
```

Comment:
Ensure that the SCSS variable `$secondary-color` is defined in your stylesheets. Without a definition, this will cause an error during compilation.